### DISCUSSION TITLE
[Question] in the navigation bar i want to have custom icons is there any way to do that (svg format)

### DISCUSSION THREAD

**Discussion by plugeit** [flet-dev/flet#1893] [Q&A] [answered]
### Question

My app requires custom designed icon is there any way to do its in svg format please help!!!

### Code sample

```python
import flet as ft

def main(page: ft.Page):

    page.title = "NavigationBar Example"
    page.navigation_bar = ft.NavigationBar(
        bgcolor=ft.colors.with_opacity(0.0, '#F5F5F5'),
        destinations=[
            ft.NavigationDestination(icon=ft.icons.HOME_OUTLINED,selected_icon=ft.icons.HOME_ROUNDED,
                label="Home"),
            ft.NavigationDestination(icon=ft.icons.COMMUTE, label="Commute"),
            ft.NavigationDestination(
                icon=ft.icons.ACCOUNT_CIRCLE_OUTLINED,
                selected_icon=ft.icons.ACCOUNT_CIRCLE_ROUNDED,
                label="Account",
            ),
        ]
    )
    page.add(ft.Text("Body!"))

ft.app(target=main, view=ft.WEB_BROWSER)
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ingdesarr1** [comment, marked as answer]
I can put a image, using the command icon_content, not icon. Maybe you can try, because you can use svg files, but not calling external, you have to copy the svg information inside. Chek this link, maybe help you

https://github.com/flet-dev/examples/blob/main/python/controls/image/svg-image.py

**Reply by plugeit** [reply]
ok i am going to try it

**Reply by plugeit** [reply]
it worked thanks very much @ingdesarr1
